ci/gpu/build.sh

```diff
@@ -48,7 +48,7 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c rapidsai/label/x
       "cudf=${MINOR_VERSION}" \
       "rmm=${MINOR_VERSION}" \
       "nvstrings=${MINOR_VERSION}" \
-      "libcumlprims=${MINOR_VERSION}" \
+      "libcumlprims=0.13" \
```

Comment:
Why is this being statically set? This should be left to be dynamic as there are builds for libcumlprims 0.14

Reply:
thanks for pointing out, now there are. will update accordingly.